### PR TITLE
removed redundant code and removed test testing redundant code

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?
Removing redundant code due to recent changes that accommodates for this.

Card: https://trello.com/c/4aJcpZAF/1812-remove-redundant-code

### How to review 
The usual

### CODE REVIEW

File: app/data_model/session_store.py [hunk 1]
```diff
@@ -77,8 +77,7 @@ def _load(self):
 
             if self._eq_session.session_data:
                 session_data = self._eq_session.session_data
```
Comment (jonnyshaw89) [inline app/data_model/session_store.py]: Can we rename this to `encrypted_session_data`?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: app/data_model/session_store.py [hunk 2]
```diff
@@ -76,11 +76,10 @@ def _load(self):
...
 
             if self._eq_session.session_data:
-                session_data = self._eq_session.session_data
-                if self._is_session_data_encrypted(session_data):
-                    session_data = StorageEncryption(self.user_id, self.user_ik, self.pepper).decrypt_data(session_data)
+                encrypted_session_data = self._eq_session.session_data
+                encrypted_session_data = StorageEncryption(self.user_id, self.user_ik, self.pepper).decrypt_data(encrypted_session_data)
 
-                self.session_data = json.loads(session_data, object_hook=lambda d: SessionData(**d))
+                self.session_data = json.loads(encrypted_session_data, object_hook=lambda d: SessionData(**d))
```
Comment (dwyeradam) [inline app/data_model/session_store.py]: After getting **encrypted_session_data** the next variable should just be **session_data**. This shows that this step is around decrypting the stored encrypted data. So line 79 you get encrypted_session_data and then on line 80 you use that to create session_data which is then used to create the **self.session_data** on line 82.

File: app/data_model/session_store.py [hunk 3]
```diff
@@ -76,11 +76,10 @@ def _load(self):
             self.user_id = self._eq_session.user_id
 
             if self._eq_session.session_data:
-                session_data = self._eq_session.session_data
-                if self._is_session_data_encrypted(session_data):
-                    session_data = StorageEncryption(self.user_id, self.user_ik, self.pepper).decrypt_data(session_data)
+                encrypted_session_data = self._eq_session.session_data
+                encrypted_session_data = StorageEncryption(self.user_id, self.user_ik, self.pepper).decrypt_data(encrypted_session_data)
```
Comment (dwyeradam) [inline app/data_model/session_store.py]: After getting encrypted_session_data the next variable should just be session_data. This shows that this step is around decrypting the stored encrypted data. So line 79 you get encrypted_session_data and then on line 80 you use that to create session_data which is then used to create the self.session_data on line 82.

**Review by dwyeradam — CHANGES_REQUESTED** [review]

File: app/data_model/session_store.py [hunk 4]
```diff
@@ -76,9 +76,9 @@ def _load(self):
             self.user_id = self._eq_session.user_id
 
             if self._eq_session.session_data:
-                session_data = self._eq_session.session_data
-                if self._is_session_data_encrypted(session_data):
```
Comment (dwyeradam) [inline app/data_model/session_store.py:80]: `def _is_session_data_encrypted(session_data)` is now no longer needed.

**Review by dwyeradam — CHANGES_REQUESTED** [review]

File: app/data_model/session_store.py (hunk 4 above)
Comment (MIslam92) [reply app/data_model/session_store.py:80]: @dwyeradam looking at my branch on Github its not there?
https://github.com/ONSdigital/eq-survey-runner/blob/eq-1812-remove-redundant-code/app/data_model/session_store.py#L78

Comment (dwyeradam) [reply app/data_model/session_store.py:80]: It's not that line that's the issue. It's the fact you still have the method in that the line was previously calling but is now no longer needed as nothing else calls it. 
https://github.com/ONSdigital/eq-survey-runner/blob/eq-1812-remove-redundant-code/app/data_model/session_store.py#L94

**Review by dwyeradam — APPROVED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Pull request merged**